### ISSUE TITLE
Bug report: github icon from the cheatsheet doesn't work

### ISSUE THREAD

**Issue by knchanu** [FortAwesome/Font-Awesome#16103]
I am trying to add the icons to my word document. All of the icons I have tried to copy+paste work just fine except for the github icons. Help?

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Could you please check that you are using the proper Font Family? It should be `Font Awesome 5 Brands`

**Comment by knchanu** [comment]
Yes, I used Font Awesome 5 Brands Font. Other icons such as linkedin, airbnb etc from the Font Awesome 5 Brands work. While I haven't tried copy+pasting all of the icons, the github and github-square icons didn't work for me. github-alt works though.

**Comment by Ondra09** [comment]
If you use .css from this site, the brand fonts have prefix fab and not fa. Example: `<i class="fab fa-linkedin">`

**Comment by sensibleworld** [comment]
Thanks for opening this issue! Here’s the simple truth… our issue tracker has been difficult to manage, so we’re doing a little housekeeping. Since it’s been a while and things have changed quite a bit in Font Awesome since this was originally posted, we’re closing it out to help keep our issue tracker focused on current work.

If this is still important for your project, please feel free to re-open or file a fresh issue with updated details. Thanks so much!
